### DISCUSSION TITLE
Welcome to openmls Discussions!

### DISCUSSION THREAD

**Discussion by franziskuskiefer** [openmls/openmls#252] [General]
## 👋 Welcome!
  We’re testing Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by raphaelrobert** [comment]
Nice, thanks for setting this up @franziskuskiefer!

I'll start with the introduction round: I'm Raphael, I started OpenMLS (it was initially called maelstrom) and I work on improving its functionality and security. I'm looking forward to working with others and hope the community will grow!

**Comment by kkohbrok** [comment]
Hi everyone!

I'm Konrad. During the course of my PhD, I joined the MLS working group, where I helped with the cryptographic analysis of the design. I recently started working on OpenMLS, where so far I have helped with making the cryptographic core more robust, as well as with other bits and bobs. I'm relatively new to Rust, but my previous experience with functional programming means I particularly enjoy using Rusts strong types to ensure that the implementation does what it should. If anyone has questions regarding the MLS protocol, I'm happy to help out!
